frontend/packages/pipelines-plugin/src/components/quicksearch/pipeline-quicksearch-utils.ts

```diff
@@ -0,0 +1,130 @@
+import i18n from 'i18next';
+import { safeLoad } from 'js-yaml';
+import * as _ from 'lodash';
+import { CatalogItem } from '@console/dynamic-plugin-sdk';
+import { coFetch } from '@console/internal/co-fetch';
+import { k8sCreate, k8sUpdate } from '@console/internal/module/k8s';
+import { TaskModel } from '../../models';
+import { TektonTaskAnnotation } from '../pipelines/const';
+import { CTALabel, TEKTONHUB } from './const';
+
+export const isSelectedVersionInstalled = (item: CatalogItem, selectedVersion: string): boolean => {
+  return item.attributes.installed === selectedVersion;
```

Comment:
nit: `attributes` look optional shall we add check
```suggestion
  return item.attributes?.installed === selectedVersion;
```